main.css

```diff
@@ -442,3 +449,86 @@ button:hover {
 .proj-cad {
   display: grid;
 }
+
+.contact-me {
+  background-color: #6070ff;
+  border: 1px solid #cfd8dc;
+  border-radius: 100px 0 0;
+  width: 372px;
+}
+
+.bg-image {
+  background-image: url(images/shapes.png);
+  background-repeat: no-repeat;
+  background-position-x: right;
+}
+
+.contact-title {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  height: 56px;
+  justify-content: center;
+  padding-top: 80px;
+}
+
+.contact-text {
+  padding-left: 24px;
+  display: flex;
+  align-items: center;
+  color: #fff;
```

Comment:
- [ ] Your text aligned left. It will better if you will use `text-align: center;` to align the text center, as required the Figma template